simple-loadflow/src/main/java/com/powsybl/loadflow/simple/ac/NewtonRaphsonParameters.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.loadflow.simple.ac;
+
+import com.powsybl.loadflow.LoadFlowParameters;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class NewtonRaphsonParameters {
```

Comment:
Should implement AbstractExtension<LoadFlowParameters>

Reply:
This class was not intended to be exposed outside of loadflow impl, but filled by LoadFlowParameters